crates/node_runtime/src/node_runtime.rs

```diff
@@ -216,6 +218,22 @@ impl NodeRuntime for RealNodeRuntime {
         Ok(installation_path.join(NODE_PATH))
     }
 
+    async fn node_environment_path(&self) -> Result<OsString> {
```

Comment:
And any idea on this function name?

Reply:
`node_path` or `node_home` ? 

Like `GOPATH` env in Go.

Reply:
> GOPATH variable (which defaults to $HOME/go on Unix and %USERPROFILE%\go on Windows) is used for the following purposes
This is a little different from `GOPATH`; here we are retrieving the `PATH` environment variable that contains the node path.

Reply:
I think that seems like a good name.